.github/workflows/linters.yaml

```diff
@@ -0,0 +1,19 @@
+---
+name: Linters
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: ludeeus/action-shellcheck@master
```

Comment:
We generally don't add random GitHub actions to our projects. I'm working on adding one in https://github.com/coreos/repo-templates/pull/72 but it needs the checks to pass here first.

Reply:
I see, will close this PR then